### ISSUE TITLE
Support all HTML attributes from MDN reference

### ISSUE THREAD

**Issue by bshankar** [vidhanio/hypertext#136]
Some attributes on [MDN attribute reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes) are missing especially event attributes like `onclick`. Another commonly used accessibility attribute is `role`.
